driver/vhci/vhci_write.c

```diff
@@ -71,6 +71,12 @@ post_select_config(pusbip_vpdo_dev_t vpdo, PURB urb)
 	USHORT	len;
 	struct _URB_SELECT_CONFIGURATION	*urb_selc = &urb->UrbSelectConfiguration;
 
+	// If ConfigurationDescriptor is NULL, the device will be set to an unconfigured state.
+	if (urb_selc == NULL || urb_selc->ConfigurationDescriptor == NULL) {
```

Comment:
Is it possible that urb_selc is NULL ? If that case will not happen, how about dropping "urb_selc == NULL" ?

Reply:
Removing. This was a leftover from my initial investigation of the crash.